README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard API
+
+A JavaScript project for the Leaderboard API implementation, using Webpack and ES6 features
+
+## Demo
+Get the live link [here](https://fabianofrank.github.io/todo-app/)
+
+## Built With
+
+- JavaScript
+- HTML and CSS
+- GIT
+- Webpack
+- API
+
+## Getting Started
+To get a local copy up and running follow these [steps](https://docs.github.com/en/repositories/creating-and-managing to-repositories/cloning-a-repository)
```

Comment:
- [ ] **REQUIRED:** It will be nice if you were more specific by providing details instructions on how install and run the project instead of redirecting the user to external links.